optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/test/java/org/optaplanner/quarkus/testdata/invalid/domain/TestdataInvalidQuarkusRoom.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.quarkus.testdata.invalid.domain;
+
+import java.util.List;
+
+import org.optaplanner.core.api.domain.variable.InverseRelationShadowVariable;
+
+public class TestdataInvalidQuarkusRoom {
```

Comment:
Can we rename this from "...Room" to "...Value", or better yet "...ShadowEntity"?

Reply:
For the existing TestdataInverseRelationValue in optaplanner-core we called it value, so let's use Value.